test/tbb/test_concurrent_queue.cpp

```diff
@@ -281,3 +281,29 @@ TEST_CASE("Test clear and dtor with TrackableItem") {
     test_tracking_dtors_on_clear<oneapi::tbb::concurrent_queue<TrackableItem>>();
     test_tracking_dtors_on_clear<oneapi::tbb::concurrent_bounded_queue<TrackableItem>>();
 }
+
+template<typename CQ, typename T>
+void test_iterator_ctor(){
+    int size = 5;
+    T vec_1(size, 0), vec_2(size, 0), vec_3(size, 0), vec_4(size, 0);
+    srand(time(0));
```

Comment:
Seems like you need `static_cast<unsigned>(time(0))` to make the Microsoft compiler happy.